Buddies.Web/src/pages/projects/[pid].tsx

```diff
@@ -54,11 +54,11 @@ const Project: React.VFC = () => {
   }, [projectId]);
 
   const addMemberToProject = async () => {
-
-    if (!(typeof projectId === 'string') || !authState) {
-      alert('Uh oh, something went wrong...');
+    if (!(typeof projectId === 'string')) {
+      alert('error');
```

Comment:
Is there a reason the `authState` check was removed? If I remember correctly it's necessary for line 62 to not trigger a TypeScript error.

Reply:
Oops, I didn't even see that. I think I might've deleted it during a merge when I wasn't paying attention, Ill put it back.